.gitignore

```diff
@@ -1,18 +1 @@
-# Build and Release Folders
-bin-debug/
-bin-release/
-[Oo]bj/
-[Bb]in/
-
-# Other files and folders
-.settings/
-
-# Executables
-*.swf
-*.air
-*.ipa
-*.apk
-
-# Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
-# should NOT be excluded as they contain compiler settings and other important
-# information for Eclipse / Flash Builder.
+test.md
```

Comment:
- [ ] In addition ti test.md the `.gitignore` should contain node_modules folder, so git wouldn't push the node modules to your Github repo, kindly add `node_modules` to your file